### PR TITLE
add fuzzer tests for config scheme

### PR DESCRIPTION
**Pull request by deads2k**
fixes https://github.com/openshift/origin/issues/7116

Adds a set of fuzzer tests for our config scheme since that is separate.  If we ever manage to collapse the schemes with a different group we won't need this anymore.

@liggitt

### CODE REVIEW

File: pkg/api/serialization_test.go [hunk 1]
```diff
@@ -412,7 +412,6 @@ func roundTrip(t *testing.T, codec runtime.Codec, originalItem runtime.Object) {
 	if err != nil {
 		if conversion.IsNotRegisteredError(err) {
 			t.Logf("%v is not registered", name)
-			return
```
Comment (deads2k) [inline pkg/api/serialization_test.go]: Yeah, this is terrifying.

**Comment by deads2k** [comment]
[test]

File: pkg/api/serialization_test.go (hunk 1 above)
Comment (liggitt) [inline pkg/api/serialization_test.go]: ...

**Comment by deads2k** [comment]
good enough?

**Comment by deads2k** [comment]
It's a test, it's strictly better than not having it.  [merge]

**Pull request merged**